app/dashboard/user/features/MenuCard.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import { useMenuCardStore } from "@/store/menu_card/MenuCardStore";
+import React, { Fragment } from "react";
+import { RowCard } from "./RowCard";
+
+export const MenuCard = () => {
+  const { setIsOpen, isOpen } = useMenuCardStore();
+  return (
+    <Fragment>
+      {isOpen ? (
+        <div
+          className="fixed top-[61.0729px] bg-black/5 w-full h-full md:hidden"
+          onClick={() => setIsOpen(false)}
+        >
+          <div
+            className="w-[80%] h-full border border-line rounded-r-[30px] bg-[#FCFCFC] px-6 py-10 space-y-5"
```

Comment:
Please use the tailwind variables for the background colour. As much as possible let's avoid using arbitrary classes